tick/survival/sccs/conv.py

```diff
@@ -0,0 +1,26 @@
+import inspect
+from tick.survival.convolutional_sccs import *
+
+class ConvSCCS_(ConvSCCS):
```

Comment:
Missing docstring. If docstring is exactly the same than ConvSCCS, just explain what are the motivations behind this class and link ConvSCCS doc page.

Reply:
you mean this ? https://x-datainitiative.github.io/tick/modules/generated/tick.survival.ConvSCCS.html#tick.survival.ConvSCCS

?

Reply:
Yeah, link to this page and explain why this class exists. In this case, the doc would be intended to future devs rather than to users, no need to do make it compilable.

Reply:
Might be a good idea to link ConvSCCS in the Stream and Batch doc too. I don't know if Sphinx handles inheritance when it comes to the doc.